test/system/components/mentoring/discussion_test.rb

```diff
@@ -24,13 +24,40 @@ class DiscussionTest < ApplicationSystemTestCase
           visit test_components_mentoring_discussion_path(discussion_id: discussion.id)
         end
 
-        assert_css "img[src='https://assets.exercism.io/tracks/ruby-hex-white.png'][alt='icon for Ruby track']"
-        assert_css "img[src='https://avatars2.githubusercontent.com/u/5337876?s=460&v=4']"\
+        assert_css "img[src='#{solution.track.icon_url}'][alt='icon for Ruby track']"
+        assert_css "img[src='#{student.avatar_url}']"\
```

Comment:
@kntsoriano Let's try and avoid using magic strings in these tests else it means we have lots of things break when we change this method later.